jni/src/test/java/com/horizen/vrfnative/VRFKeyPairTest.java

```diff
@@ -1,127 +1,127 @@
+//            -21, -53, -10, 108, 72, -87, 121, -40, 45, -126, 104, -95, 117, -45, 29, -40, -58, -83, -5, 127, 108, -18,
+//            -95, 75, -88, -40, 116, -13, 78, 28, -6, -92, -63, 1, 0, 0, -120, -87, 92, -58, 73, -66, 17, 4, 76, -91, 27,
+//            24, -51, -15, -20, -128, -8, -67, 54, -76, 22, 58, -28, 19, 29, 121, -95, 11, -80, 25, 112, -104, -37, 47,
+//            18, -104, 43, -69, -85, 65, 21, -67, -63, 37, 65, -65, -30, 95, -1, -30, -46, 47, -48, 122, -102, -124, 52,
+//            95, -25, 122, 30, 21, 113, -41, 99, -74, -60, 107, -120, -63, -16, -54, -115, 64, -17, 32, 15, 84, 117, -62,
+//            -116, 111, 62, -13, 86, 66, 15, 27, -86, -42, 29, -127, 60, -69, 0, 0, -21, 63, -49, 85, 115, -99, 49, -55,
+//            37, 114, 29, 19, 64, -71, 4, 46, -111, 108, -121, -32, -127, 49, -47, 33, 119, 108, -109, 90, 60, -5, 118,
+//            1, -28, 87, 107, 63, 96, 16, 113, 110, -61, 66, -3, -126, -75, -80, 122, 97, -98, 49, -94, -11, -32, -13,
+//            88, -12, -127, -52, -33, -63, -112, 8, 64, 58, 49, -4, 86, -76, 40, 56, 28, 120, 92, 14, -112, -53, 8, 31,
+//            -41, -77, -73, -80, 50, -62, 99, 94, -67, 15, -110, 26, -41, 27, -126, 123, 0, 0
+//        };
+//
+//        byte[] vrfOutputBytes = {
+//            -126, 18, -112, -103, 101, 88, -117, -21, 7, -72, -74, -128, -128, -99, -107, 31, -48, -44, 44, 38, 121, 28,
+//            -33, -88, -74, -68, 66, 26, -100, 115, -95, -98, -50, -57, -112, -81, 16, -72, -118, -58, -74, 65, 90, -96,
+//            -75, -56, -4, 44, 104, 44, 85, -99, -31, -99, 31, 11, -78, 118, -108, 96, -30, 55, 56, 30, -45, -109, -48,
+//            -9, 101, 30, -95, -15, -52, -72, -53, -97, -22, 120, 115, -54, 97, 16, -30, -54, 3, -11, 36, -97, -70, -46,
+//            42, -43, 11, 104, 1, 0
+//        };
+//
+//        try
+//        (
+//            VRFSecretKey sk = VRFSecretKey.deserialize(skBytes);
+//            VRFKeyPair keyPair = new VRFKeyPair(sk);
+//            FieldElement message = FieldElement.deserialize(messageBytes);
+//            VRFProof proof = VRFProof.deserialize(proofBytes);
+//            FieldElement expectedVrfOutput = FieldElement.deserialize(vrfOutputBytes)
+//        )
+//        {
+//            assertNotNull("sk deserialization must not fail", sk);
+//            assertNotNull("message deserialization must not fail", message);
+//            assertNotNull("proof deserialization must not fail", proof);
+//            assertNotNull("expectedVrfOutput deserialization must not fail", sk);
+//
+//            try(FieldElement vrfOutput = keyPair.getPublicKey().proofToHash(proof, message))
+//            {
+//                assertNotNull("VRF Proof verification and VRF Output computation has failed.", vrfOutput);
+//
+//                // Check vrfOutput == expectedVrfOutput
+//                assertEquals("vrfOutput and expectedVrfOutput must be equal", vrfOutput, expectedVrfOutput);
+//            }
+//        }
+//    }
+//
+//    @Test
```

Comment:
Don't see any usage of hardcoded data here. Maybe we can keep it ?

Reply:
Yes, this test is working, I've uncommented it